validation/genesis/genesis-predeploy_test.go

```diff
@@ -35,6 +53,8 @@ func TestGenesisPredeploys(t *testing.T) {
 			})
 		}
 	}
+
+	// TODO cleanup temporaryOptimismDir
```

Comment:
Needs a github issue I think (if you are not planning to address in this PR)

Reply:
I will address that on the feature branch PR before merging that one.